utils/sv_easing_functions.py

```diff
@@ -328,35 +328,46 @@ def BounceEaseInOut(p, settings=defaultBounceSettings):
 
 
 easing_dict = {
+    # LINEAR
```

Comment:
Changing the dict num->func mapping might make sense logically but it breaks existing blends that use the easing node